p2p/src/main/java/bisq/network/p2p/peers/getdata/messages/PreliminaryGetDataRequest.java

```diff
@@ -28,24 +28,28 @@
 
 import com.google.protobuf.ByteString;
 
+import java.util.Optional;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 import lombok.EqualsAndHashCode;
 import lombok.Value;
 import lombok.extern.slf4j.Slf4j;
 
-import org.jetbrains.annotations.NotNull;
+import javax.annotation.Nullable;
 
 @Slf4j
 @EqualsAndHashCode(callSuper = true)
 @Value
 public final class PreliminaryGetDataRequest extends GetDataRequest implements AnonymousMessage, SupportedCapabilitiesMessage {
     private final Capabilities supportedCapabilities;
 
-    public PreliminaryGetDataRequest(int nonce,
-                                     @NotNull Set<byte[]> excludedKeys) {
-        this(nonce, excludedKeys, Capabilities.app, Version.getP2PMessageVersion());
+    public PreliminaryGetDataRequest(int nonce, Set<byte[]> excludedKeys) {
+        this(nonce,
+                excludedKeys,
+                Version.VERSION,
+                Capabilities.app,
+                Version.getP2PMessageVersion());
     }
 
```

Comment:
extra white space

Reply:
Where? the 2 linebreaks between constr. and comment lines are intended.